contrib/vdirsyncer.plist

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+  <dict>
+    <!-- click requires a proper UTF-8 locale -->
+    <key>EnvironmentVariables</key>
+    <dict>
+           <key>LANG</key>
+           <string>en_US.UTF-8</string>
+           <key>LC_ALL</key>
+           <string>en_US.UTF-8</string>
```

Comment:
No, to the locale of the system vdirsyncer runs on.

On 26 April 2016 09:28:34 CEST, benjaminfrank <notifications@github.com> wrote:
>> @@ -0,0 +1,34 @@
>> +<?xml version="1.0" encoding="UTF-8"?>
>> +<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN"
>"http://www.apple.com/DTDs/PropertyList-1.0.dtd">
>> +<plist version="1.0">
>> +  <dict>
>> +    <!-- click requires a proper UTF-8 locale -->
>> +    <key>EnvironmentVariables</key>
>> +    <dict>
>> +           <key>LANG</key>
>> +           <string>en_US.UTF-8</string>
>> +           <key>LC_ALL</key>
>> +           <string>en_US.UTF-8</string>
>
>So basically a user needs to set the locale to the one used for
>creating the entries/the server uses?
>
>---
>You are receiving this because you modified the open/close state.
>Reply to this email directly or view it on GitHub:
>https://github.com/pimutils/vdirsyncer/pull/435/files/89181d1cb25be28a22d3fe436ab3b5b79c1487b0#r61039109

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.